### ISSUE TITLE
Small quibble with activating the touch and swipe on graphs to show the details pop-ups

### ISSUE THREAD

**Issue by DigiH** [theengs/app#40]
Might be iOS only? Needs verification on Android.

When I want to get the widget view graphs details pop-ups I would expect the pop-ups to appear for the initial **touch down** event, then being able to slide along for the pop-ups to show the history details.

The initial pop-ups, and it's related vertical time line indicator, only show up with by first **touch up** event though. 

Making it a bit unexpected and awkward with **touch down - wondering - touch up - now I see them - touch down again and slide** to get what I wanted.

Same when the pop-ups and time line indicator is already showing, **touch down** at a new location doesn't do anything, as would be expected, only **touch up** again makes the indicator and pup-up values jump to the newly selected position.

**Comment by emericg** [comment]
You mean using the press and not the click on mobile? Why not. I've just tried, but that makes it weird between touching, dragging the graph bar and swiping to the next screen. I'll have to see.
On desktop it really doesn't make sense.

**Comment by emericg** [comment]
Actually, if the drag area is merged with the press area (and the screen swipe is disabled by a press) it feels pretty good to use...

**Comment by DigiH** [comment]
Yes, was only talking about **touch ui** devices, Android and iOS/iPadOS.
